app/assets/stylesheets/opportunities.scss

```diff
@@ -0,0 +1,45 @@
+@import "foundation_and_overrides";
+
+.vol-panel, .opportunity-info {
+  border: 5px solid #FF851B;
+  color: white;
+  background-color: #496d7b;
+  margin: 2%;
+  padding: 2%;
+  text-align: center;
+}
+
+header {
+  margin: 5%;
+}
+
+h1, h2 {
+  color: white;
+  text-align: center;
+}
+
+.join-button {
+  background-color: #FF851B;
+  color: white;
+}
+
+#org-name {
+  a {
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.